src/styles/sidebar/components/hypothesis-app.scss

```diff
@@ -20,6 +20,15 @@
   &__content {
     @include layout.sidebar-content;
   }
+
+  // FIXME: Temporary affordance for the Notebook view to override some
```

Comment:
Like it says, this is hyper-temporary. Getting `HypothesisApp` sorted out is high on my TODO list.